docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #82.            max_attempts = int(np.ceil(1.0 / decay[i])) + 1
</pre><p>Since <code>max_attempts</code> only depends on <code>decay</code> then you might as well create a 1-D array after you create <code>decay</code> :</p><p> </p><pre class="ql-syntax" spellcheck="false">max_attempts = np.ceil(1.0 / decay).astype(np.int64) + 1   
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
You are right! I will change this. Thanks for the suggestion!